tests/test_vcert.rb

```diff
@@ -10,7 +10,7 @@
 TPPUSER = ENV['TPPUSER']
 TPPPASSWORD = ENV['TPPPASSWORD']
 TPPZONE = ENV["TPPZONE"]
-TRUST_BUNDLE = "/tmp/chain.pem"
+TRUST_BUNDLE = ENV["TRUST_BUNDLE"]
```

Comment:
In over products we use hardcoded value for tests

Reply:
Everywhere in Jenkinsfile TRUST_BUNDLE is set from variables